LuizWeitz.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@LuizWeitz"
+project: "E-reader using Raspberry Pi Pico"
+---
+
+# E-reader With Raspberry Pi Pico
+
+## Summary
+
+E-reader using Raspberry Pi Pico with E-Ink technology screen with rechargeable battery that fits in the palm of the hand, through which I intend to expand my adventures in the world of reading.
+
+## Plan
```

Comment:
Are you following a guide or had some inspo from the internet? Please give us some links if you do :)

Reply:
Hi, I have follow this project https://github.com/joeycastillo/The-Open-Book to get some ideas and I also did some research on the internet regarding the components, for example the screen I will follow the manufacturer's documentation.

Reply:
Can you please add that link to your planning section? thanks :)

Reply:
Hi, yes😃